pandas/tests/indexing/test_iloc.py

```diff
@@ -388,52 +388,61 @@ def test_iloc_getitem_frame(self):
 
         result = df.iloc[2]
         with catch_warnings(record=True):
+            filterwarnings("ignore", "\\n.ix", DeprecationWarning)
```

Comment:
use ignore_ix?

Reply:
That filters the warning for the entire test. I was trying to exactly match the tests previous intent, of only filtering within the `catch_warnings` blocks.